### ISSUE TITLE
docker build fails

### ISSUE THREAD

**Issue by rcemper** [intersystems-community/zpm-dockerhub#35]
Created4.4s 1  Creating                                                                                      6.5s
[+] Running 2/3m-dockerhub-iris-1
 ✔ Network zpm-dockerhub_default
                    Created4.4s atform (linux/arm64/v8) **does not match the detected host platform (linux/amd64/v2)** and no specific pla
[+] Running 2/3m-dockerhub-iris-1
 ✔ Network zpm-dockerhub_default

**Comment by daimor** [comment]
This repository not supposed to be used that way, it's only to produce community images

**Comment by evshvarov** [comment]
No, it is important to have it on OEX as everyone who uses it could know of
such community images being created and how it works.
Also people often send PR to the workflow when new IRIS versions appear to
bake new community images
